src/main/webapp/js/common.js

```diff
@@ -450,33 +450,37 @@ function sortByDiff(a, b) {
  */
 function getPointValue(s, ditchZero) {
     var s0 = s;
+    var baseValue = 100;
+    
     if (s0.lastIndexOf('<') !== -1) {
         s0 = s0.substring(0, s0.lastIndexOf('<'));
         s0 = s0.substring(s0.lastIndexOf('>') + 1);
     }
     
```

Comment:
Looked through all the files that used table sorting code.

Because of line 308 in `common.js`, all elements being compared are text and no where do other files using `sortByPoint` or similar functions directly. Hence it is safe to remove the html unwrapping code.